### PR TITLE
Remote storage with Flysystem

### PR DESCRIPTION
**Pull request by jonathanjfshaw**
I was raving to Janez about how easy it is to do remote storage on S3 or Dropbox with Flysystem and file fields on media or content entities, so he suggested I contribute that to this guide.

### CODE REVIEW

File: remote_storage/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+If you upload a file in Drupal using that file field, it will then be stored on your remote location, not on the web server where Drupal runs.
+
+When the file field is rendered by Drupal (e.g. because the content or media entity having that file field is requested by a site visitor) then the file will be served up to your visitor, exactly as if it were stored on your server. All the Drupal field formatters, theming, etc. work exactly as normal.
+
+Flysystem's file access control
+--------------------------------------------------
+Flysystem storage schemes works like private files in drupal 8.
+
+It hides the actual link of the to file and delivers it via a Drupal generated path. Thus it restricts access to the original files and let other modules provide access control to the file.
```
Comment (jonhattan) [inline remote_storage/README.md]: Wrong wording: "of the to file"

Comment (jonathanjfshaw) [inline remote_storage/README.md]: Thanks! That prompted me to improve that whole paragraph.

**Comment by slashrsm** [comment]
Merged. Thanks!

**Comment by davereid** [comment]
@jonathanjfshaw Thanks for your contribution! I fixed up your markdown links and moved things around between a general 'file systems' page from the 'flysystem' page. Hopefully those changes moving things around are ok with you.

**Comment by jonathanjfshaw** [comment]
Yeah, I thought something like that should probably happen as the content grew. Thanks Dave :)

**Pull request merged**